WooCommerce/src/androidTest/kotlin/com/woocommerce/android/screenshots/util/DataClasses.kt

```diff
@@ -0,0 +1,78 @@
+package com.woocommerce.android.screenshots.util
+
+// While the test data for reviews is obtained from
+// "mappings/jetpack-blogs/wc/reviews/products_reviews_all.json",
+// this file does not contain human-readable product descriptions,
```

Comment:
Nitpick

```suggestion
// that file does not contain human-readable product descriptions,
```

Reading "this file" makes me think of the file were the comment is, not the one just mentioned.

Reply:
Good catch, thanks!